src/ServiceControl/App.config

```diff
@@ -22,7 +22,7 @@ These settings are only here so that we can debug ServiceControl while developin
     <!--<add key="ServiceControl/TransportType" value="SQLServer" />-->
 
     <add key="ServiceControl/PersistenceType" value="RavenDB" />
-    <add key="ServiceControl/DataSpaceRemainingThreshold" value="10" />
+    <add key="ServiceControl/DataSpaceRemainingThreshold" value="20" />
```

Comment:
Seems like we could just remove this if we're going to set it to default value?

Looks like don't have this in the other app.config files, so it seems fine to remove.